projects/demo/src/modules/components/input/examples/10/index.html

```diff
@@ -54,5 +54,10 @@ <h4>
     <code>autocomplete="new-password"</code>
     We could use
     <code>new-password</code>
-    , which should tell the browser, that this is like a new password and autofill is therefore not useful here.
+    which should tell the browser, that this is like a new password and autofill is therefore not useful here.
+</p>
+
+<p>
+    Another way you can use this combination of html attributes:
+    <code>&lt;input role="presentation" autocomplete="off" /&gt;</code>
```

Comment:
nit:

Provide source-link where our users can read more about it, please

Reply:
Is it okey?
https://gist.github.com/niksumeiko/360164708c3b326bd1c8?permalink_comment_id=3793032

I can't find official solution, but I check `<input role="presentation" autocomplete="off" />` it's works

Reply:
This comment ?

https://gist.github.com/niksumeiko/360164708c3b326bd1c8?permalink_comment_id=3925346#gistcomment-3925346

Reply:
Let add link
Someday, it stops working.
At least, our users can open this link to find new solution :D